src/test/java/org/reactome/release/cosmicupdate/COSMICUpdateUtilTest.java

```diff
@@ -0,0 +1,236 @@
+package org.reactome.release.cosmicupdate;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.SchemaClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class COSMICUpdateUtilTest
+{
+
+	private static final String IDENTIFIER_COSF1234 = "COSF1234";
+	private static final String IDENTIFIER_1234 = "1234";
+	private static final long DBID_1235 = 1235L;
+	private static final long DBID_1234 = 1234L;
+	private static final String IDENTIFIER_COSM1111 = "COSM1111";
+	private static final String IDENTIFIER_44444 = "44444";
+	private static final String IDENTIFIER_5678 = "5678";
+	private static final String IDENTIFIER_COSF6321 = "COSF6321";
+	private static final String IDENTIFIER_COSF7890 = "COSF7890";
+	private String COSMICFusionExportFile = null;
+	private String COSMICMutationTrackingFile = null;
+	private String COSMICMutantExportFile = null;
+	
+	private static final String COSMIC_GENOMIC_MUTATION_ID = "GENOMIC_MUTATION_ID";
+	private static final String COSMIC_MUTATION_ID = "MUTATION_ID";
+	private static final String COSMIC_LEGACY_MUTATION_ID = "LEGACY_MUTATION_ID";
+
+	private static final String MUTANT_HEADER =  COSMIC_LEGACY_MUTATION_ID + "\t" +  COSMIC_MUTATION_ID+ "\t" + COSMIC_GENOMIC_MUTATION_ID + "\n"; 
+	
+	@Before
+	public void setup() throws IOException
+	{
+		// need to create temp files to read.
+		createTestCosmicFusionExport();
+		createTestCosmicMutantExport();
+		createTestCosmicMutationTracking();
+		
+	}
+
+
+	private void createTestCosmicMutationTracking() throws IOException
+	{
+		Path pathToMutationTracking = Files.createTempFile("mutationTracking", "csv");
+		this.COSMICMutationTrackingFile = pathToMutationTracking.toString();
+		Files.writeString(pathToMutationTracking, MUTANT_HEADER);
+		Files.writeString(pathToMutationTracking, "1234\t1234\tCOSV2323232\n", StandardOpenOption.APPEND);
```

Comment:
Could the "IDENTIFIER_1234" constant be used here instead of hard-coding the "1234" value?